README.md

```diff
@@ -300,20 +353,20 @@ function updateResultsCount(count, searchTerm) {
 updateResultsCount(10, "apple")
 ```
 
-**Beware** that if you trigger multiple announcements that contain the same message, some screen readers only announce the first one and ignore subsequent ones.
+**Beware:** if you trigger multiple announcements that contain the same message, some screen readers only announce the first one and ignore subsequent ones.
 To avoid this announced-once issue, please ensure that each alert message is different.
 
 
-**Beware** that the container that has `aria-live` must exist on the DOM before the content within is modified.
+**Beware:** the container that has `aria-live` must exist on the DOM before the content within is modified.
 If `aria-live` container is added along with new content it will not be picked up by some screen readers.
 
 A related attribute is [aria-atomic](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#additional_live_region_attributes) which indicates to the screen reader whether it should announce:
 - the entire block when anything changes, or
 - the individual change by itself
 
-# Collapsible/Expandable Content
+## <a name="expandableContent"></a>Collapsible/Expandable Content
 
-A common design pattern is to have a button that reveals some section, like an accordion.
+A common design pattern is to have a button that reveals some section, like an accordion or a modal.
```

Comment:
```suggestion
A common design pattern is to have a button that reveals some section, like an accordion.
```
When opening a modal, the focus should move which ignores this status announcement